### PR TITLE
Additional logs on debug level - improvements

### PR DESCRIPTION
**Pull request by jdziurzynski**
KNTK - 262

Regarding to https://github.com/kentik/terraform-provider-kentik-synthetics/pull/86#issuecomment-1048658313
https://github.com/kentik/terraform-provider-kentik-cloudexport/pull/54#discussion_r812712734

### CODE REVIEW

File: synthetics/data_source_agent.go [hunk 1]
```diff
@@ -21,11 +20,11 @@ func dataSourceAgent() *schema.Resource {
 }
 
 func dataSourceAgentRead(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
-	tflog.Debug(ctx, fmt.Sprintf("Kentik API request - read agent with ID: %s\n", d.Get(idKey).(string)))
+	tflog.Debug(ctx, "Get synthetic Kentik API request", "ID", d.Get(idKey).(string))
 	resp, httpResp, err := m.(*kentikapi.Client).SyntheticsAdminServiceAPI.
 		AgentGet(ctx, d.Get(idKey).(string)).
 		Execute()
-	tflog.Debug(ctx, fmt.Sprintf("Kentik API response - read:\n%s\n", httpResp.Body))
+	tflog.Debug(ctx, "Get synthetic Kentik API response", resp)
```
Comment (danielfurman) [inline synthetics/data_source_agent.go:27]:
```suggestion
	tflog.Debug(ctx, "Get synthetic Kentik API response", "response", resp)
```
Could you show if the output of the log is correct? I think the Debug() API requires key-value pairs after the message.

In other places too.

File: synthetics/data_source_agent.go [hunk 2]
```diff
@@ -21,11 +20,11 @@ func dataSourceAgent() *schema.Resource {
 }
 
 func dataSourceAgentRead(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
-	tflog.Debug(ctx, fmt.Sprintf("Kentik API request - read agent with ID: %s\n", d.Get(idKey).(string)))
+	tflog.Debug(ctx, "Get synthetic Kentik API request", "ID", d.Get(idKey).(string))
```
Comment (danielfurman) [inline synthetics/data_source_agent.go:23]:
```suggestion
	tflog.Debug(ctx, "Get Agent Kentik API request", "ID", d.Get(idKey).(string))
```
The name of the resource is Agent.
In other places too.

File: synthetics/data_source_tests.go [hunk 3]
```diff
@@ -38,9 +37,9 @@ func dataSourceTests() *schema.Resource {
 }
 
 func dataSourceTestsRead(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
-	tflog.Debug(ctx, "Kentik API request - read tests")
+	tflog.Debug(ctx, "List synthetic Kentik API request")
```
Comment (danielfurman) [inline synthetics/data_source_tests.go:40]:
```suggestion
	tflog.Debug(ctx, "List Tests Kentik API request")
```
The name of the resource is Test...
In other places too.

**Review by danielfurman — CHANGES_REQUESTED** [review]

File: synthetics/data_source_agent.go (hunk 1 above)
Comment (danielfurman) [reply synthetics/data_source_agent.go]: Done

File: synthetics/data_source_agent.go (hunk 2 above)
Comment (danielfurman) [reply synthetics/data_source_agent.go]: Done

File: synthetics/data_source_tests.go (hunk 3 above)
Comment (danielfurman) [reply synthetics/data_source_tests.go]: Done

**Review by danielfurman — APPROVED** [review]

**Review by mmac-m3a — APPROVED** [review]

**Pull request merged**